2-multitenant/modules/env_baseline/README.md

```diff
@@ -15,6 +15,7 @@ The following resources are created:
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
+| apps | Applications | <pre>map(object({<br>    ip_address_names = list(string)<br>  }))</pre> | n/a | yes |
```

Comment:
Here or in a higher level README.md, define "app".

Some things to include in the definition:
- An "App" is high-level grouping of related services and workloads.
- That and App may include multiple namespaces, and also multiple team scopes.
- New apps are created infrequently. 
-  That Cymbal Bank example is one App, called `cymbal-bank`.
- That different Apps don't share IP addresses.
- A multi-tenant cluster can contain multiple apps.

Reply:
Sure - I'll add to the 2-multitenant README for this PR.  We'll need substantial changes to most of the other stages before the concept is "global".